### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by Bowt13**
sorry for being partially on the wrong branch.
edit signup. Add uppercase in GameList. BattleArena updated so that the animation is fluid.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "material-ui": "next",
     "material-ui-icons": "^1.0.0-beta.36",
+    "nodemon": "^1.17.2",
```
Comment (nsch1) [inline package.json:8]: Doesn't need to be in dependencies for react app.

File: package.json [hunk 2]
```diff
@@ -20,6 +21,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
+    "eject": "react-scripts eject",
+    "dev": "nodemon ."
```
Comment (nsch1) [inline package.json:25]: Doesn't need to be here for react app, already have start.

File: src/actions/users.js [hunk 3]
```diff
@@ -39,10 +39,10 @@ export const login = (email, password) => (dispatch) =>
     	}
     })
 
-export const signup = (email, password) => (dispatch) =>
+export const signup = (email,username, password) => (dispatch) =>
 	request
 		.post(`${baseUrl}/users`)
-		.send({ firstName: email, lastName: email, email, password })
+		.send({ username: username, email, password })
```
Comment (nsch1) [inline src/actions/users.js:45]: Nice work!

File: src/containers/Game/BattleArena.js [hunk 4]
```diff
@@ -47,53 +47,57 @@ timerAttackType = (x, y) => {
...
-      <div className={this.props.background}>
-        <HealthBar side='left' percentage={this.calulatePercentage(0)}/>
-        <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
-        {this.state.playerStatus === 'idle' &&
-        <div>
-          <Player side='left' character={this.props.player1} imgSrc={require (`../../img/classes/${this.props.player1}-${this.state.playerStatus}.gif`)} status={this.state.playerStatus} attackType={this.state.attackType}/>
-          <Player side='right'character={this.props.player2} imgSrc={require (`../../img/classes/${this.props.player2}-${this.state.playerStatus}.gif`)} status={this.state.playerStatus} attackType={this.state.attackType}/>
-          {this.timerPlayerState('move', 450)}
+      <div>
+      {this.props.player1 !== 'no' && this.props.player !== 'no' &&
```
Comment (nsch1) [inline src/containers/Game/BattleArena.js:51]: Why does this use player1 and player when the rest uses player1 & player2?

File: src/containers/Page.js [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
 
-
-//STYLING
-
-
-//CLASSES
-
-
-export default class Page extends PureComponent {
+export class Page extends PureComponent {
```
Comment (nsch1) [inline src/containers/Page.js:11]: What page is this?

File: src/containers/Page.js [hunk 6]
```diff
@@ -26,12 +25,44 @@ export default class Page extends PureComponent {
...
+
+
   render() {
+    const {game} = this.props
     return (
+      <div>
+      {game === null &&
       <div className='game'>
-        <BattleArena background='forest' player1='fighter' player2='mage'/>
+        <BattleArena background='forest' player1='no' player2='no'/>
```
Comment (nsch1) [inline src/containers/Page.js:42]: Can't you get the players from redux state?

File: src/containers/Page.js [hunk 7]
```diff
@@ -26,12 +25,44 @@ export default class Page extends PureComponent {
...
         <MenuBar/>
       </div>
+      }
+      {game !== null &&
```
Comment (nsch1) [inline src/containers/Page.js:46]: You can just use game, null evaluates to false.

**Review by nsch1 — APPROVED** [review]
Looks good! Just left a few questions about things in the review.

File: package.json (hunk 1 above)
Comment (Bowt13) [reply package.json:8]: Will fix that.

File: package.json (hunk 2 above)
Comment (Bowt13) [reply package.json:25]: Will also fix that.
Thnx Good find.

File: src/containers/Game/BattleArena.js (hunk 4 above)
Comment (Bowt13) [reply src/containers/Game/BattleArena.js:51]: forgot to add 2 fixed it just now.

File: src/containers/Page.js (hunk 5 above)
Comment (Bowt13) [reply src/containers/Page.js:11]: temporarily name. should be gameScreen or something.

File: src/containers/Page.js (hunk 6 above)
Comment (Bowt13) [reply src/containers/Page.js:42]: yes i can and yes i will.

**Pull request merged**